src/checkstyle/checkstyle.xml

```diff
@@ -55,10 +55,13 @@
         <!-- Class Design -->
         <module name="com.puppycrawl.tools.checkstyle.checks.design.FinalClassCheck" />
         <module name="com.puppycrawl.tools.checkstyle.checks.design.InterfaceIsTypeCheck" />
-        <!-- TODO: re-enable when ready -->
-        <!--
-        <module name="com.puppycrawl.tools.checkstyle.checks.design.HideUtilityClassConstructorCheck" />
-        -->
+        <module name="com.puppycrawl.tools.checkstyle.checks.design.HideUtilityClassConstructorCheck" >
+            <module name="SuppressionCommentFilter">
+                <property name="offCommentFormat" value="HideUtilityClassConstructorCheck OFF"/>
```

Comment:
You can't really modify the module definition, if we ever want to use the one upstream.